ProjFS.Mac/PrjFSKext/PrjFSKext/VirtualizationRoots.cpp

```diff
@@ -34,12 +35,22 @@ struct VirtualizationRoot
     char                        path[PrjFSMaxPath];
 };
 
+struct UsedMountPoint
```

Comment:
Yeah, I left this one generic because I anticipate ending up using this data structure for other purposes. For example, instead of `VirtualizationRoot_VnodeIsOnAllowedFilesystem()`, we could early-out of kauth callbacks based on looking up `vnode_mount(currentVnode)` in this data structure. It would need to be something more efficient than an array in that case though.